docs/application/dotnet/guides/uix/sticker.md

```diff
@@ -0,0 +1,351 @@
+# Sticker
+
+
+The Sticker feature provides users with fun experiences to share sticker information. The Sticker applications are composed of a [Sticker provider application](#provider_application) and [Sticker consumer applications](#consumer_application). The Sticker provider creates stickers and you can create your own Sticker consumer applications that use the stickers created by the Sticker provider. The Sticker provider application can store sticker details such as URI, keyword, and group name in the Sticker database. The Sticker consumer applications can retrieve sticker information using the Sticker API.
```

Comment:
```suggestion
The Sticker feature provides users with fun experiences to share sticker information. The Sticker applications are composed of a [Sticker provider application](#provider_application) and [Sticker consumer applications](#consumer_application). The Sticker provider creates stickers, and you can create your own Sticker consumer applications that use the stickers created by the Sticker provider. The Sticker provider application can store sticker details such as URI, keyword, and group name in the Sticker database. The Sticker consumer applications can retrieve sticker information using the Sticker API.
```

Reply:
Dear @omar-srbd,

Inhong who is the author is absent due to attending the training session, and I can't modify in his forked repository.
I'll upload the separate PR to apply your suggestion.
Would you please review first except this suggestion?

Reply:
@imfine98, thanks for informing the matter.
Since this is a very minuscule change and the author is absent I will approve this PR and make the suggested change by myself in another PR in the near future.

Reply:
@omar-srbd,
Thank you very much.